app/assets/javascripts/application.js

```diff
@@ -34,62 +34,13 @@
 //= require contact_us
 //= require sign_up
 //= require reviews
+//= require sidebar
 
 var ready = function() {
-	var toggleSpeed = 125;
-
-	// Attatches navbar-toggle button to sidebar
-	$('.navbar-toggle').click(function() {
-		$('aside').toggle('slide', {
-			direction: 'left'
-		}, toggleSpeed);
-	});
-
-	// expands user-account options in sidebar on click
-	$("a#user-account").click(function() {
-		$(".col-secondary").toggle(toggleSpeed);
-	});
-
-	$(document).mousedown(function(e) {
-		// if click outside of sidebar, and window length is less than 850px, retract sidebar.
-		if (!$("aside").is(e.target) && $("aside").has(e.target).length === 0 && $(window).width() < 850 && !$(".navbar-toggle").is(e.target) && $(".navbar-toggle").has(e.target).length === 0) {
-			$("aside").hide('slide', toggleSpeed);
-		}
-	});
-
-	// retracts sidebar if esc key is pressed
-	$(document).keydown(function(e) {
-		if (e.which === 27 && $(window).width() < 850) {
-			$("aside").hide('slide', toggleSpeed);
-		}
-	});
-
 	$("#close-notice, #close-alert").click(function() {
 		$(this).parent().slideUp();
 	});
 
-	$("#search-query").focus(function() {
-		if ($("#search-query").val() == "") {
-			$(".nav-row").each(function() {
-				if (!($(this).hasClass("search-row"))) {
-					$(this).slideUp();
-				}
-			});
-
-			$(".submit-row").slideDown();
-		}
-	});
-
-	$("#search-query").blur(function() {
-		if ($("#search-query").val() == "") {
-			$(".nav-row").each(function() {
-				$(this).slideDown();
-			});
-
-			$(".submit-row").slideUp();
-		}
-	});
-
 	$("#word-cloud-switch").bootstrapSwitch({
```

Comment:
Alan actually meant the word cloud switch. That and the doge functionality have now been removed.